### ISSUE TITLE
Image width and height not working

### ISSUE THREAD

**Issue by speedlightwp** [DavBfr/dart_pdf#272]
pdf: ^1.6.0

  ```
  final pw.Document doc = pw.Document();

    Uint8List bytes = (await rootBundle.load('assets/logo.jpg')).buffer.asUint8List();
   
    final image = PdfImage.file(
      doc.document,
      bytes: bytes,
    );

   pw.Header(
      level: 0,
      child: pw.Row(
             mainAxisAlignment: pw.MainAxisAlignment.spaceBetween,
             children: <pw.Widget>[
                    pw.Text('Title', textScaleFactor: 1.2),
                    pw.Image(image, width: 50, height: 50),
             ]
       )
   ),

```

**pw.Image(image, width: 50, height: 50),**
Unable to adjust the width and height. The image is still at default size.

**Comment by DavBfr** [comment]
You're right, width and height are not implemented. I will fix that.

**Comment by DavBfr** [comment]
Fixed in version 1.6.1
